content/projects/gsoc/2020/projects/external-fingerprint-storage.adoc

```diff
@@ -41,41 +41,63 @@ image:/images/post-images/gsoc-external-fingerprint-storage-for-jenkins/overview
 
 * Build the Fingerprint External Storage API Plugin to track the flow of fingerprints across the entire CI/CD flow.
 
+=== Trying it out!
+
+If you are a Jenkins user, consider trying out the link:https://plugins.jenkins.io/redis-fingerprint-storage/[Redis
+Fingerprint Storage Plugin].
+We appreciate you trying out the plugin, and welcome any suggestions, feature requests, bug reports, etc.
+
+Feel free to reach out to us for any questions, feedback, etc. on the project’s Gitter Channel or the Jenkins Developer
```

Comment:
Could you include a link to the specific Gitter channel?  I'm not sure which channel is used.
```suggestion
Feel free to reach out to us for any questions, feedback, etc. on the project’s Gitter Channel or the Jenkins Developer
```